cache/in-memory/src/model/message.rs

```diff
@@ -12,31 +12,60 @@ use twilight_model::{
     id::{ChannelId, GuildId, MessageId, RoleId, UserId, WebhookId},
 };
 
+/// Represents a cached [`Message`].
+///
+/// [`Message`]: twilight_model::channel::Message
 #[derive(Clone, Debug, Eq, PartialEq, Serialize)]
 pub struct CachedMessage {
-    pub id: MessageId,
+    /// For rich presence chat embeds, the activity object.
     pub activity: Option<MessageActivity>,
+    /// For interaction responses, the ID of the interaction's application.
     pub application: Option<MessageApplication>,
+    /// Attached files.
     pub attachments: Vec<Attachment>,
+    /// ID of the message author.
+    ///
+    /// If the author is a webhook, this is its ID.
```

Comment:
Is this true considering the webhook_id field?

Reply:
yes, it's very strange. is it worth it to change this to  UserOrWebhook? probably not